server/server.go

```diff
@@ -144,3 +150,18 @@ func psmdb(vs *pbVersion.VersionResponse, deps Deps, req *pbVersion.ApplyRequest
 
 	return nil
 }
+
+func validate(req *pbVersion.ApplyRequest) error {
```

Comment:
i don't like the function name at all, because it's not validating incoming request which is missleading

Reply:
I do not know good name
https://github.com/Percona-Lab/percona-version-service/pull/31/commits/19c9a2e7ca0fec8ad26113b4c75cd747b6f701d6
If you have any suggestions let me know

Reply:
i think this one is good enough =)